pkg/action/build-iso.go

```diff
@@ -183,6 +183,19 @@ func (b BuildISOAction) createEFI(rootdir string, isoDir string) error {
 		b.cfg.Logger.Errorf("Failed writing grub.cfg: %v", err)
 		return err
 	}
+	// Ubuntu efi searches for the grub.cfg file under /EFI/ubuntu/grub.cfg while we store it under /boot/grub2/grub.cfg
+	// workaround this by copying it there as well
+	// TODO: Should we symlink all the distros folders to our single grub.cfg?
+	err = utils.MkdirAll(b.cfg.Fs, filepath.Join(isoDir, "EFI/ubuntu/"), constants.DirPerm)
```

Comment:
so every distro will have an `ubuntu` directory now? Maybe we should try the symlink idea instead. That fix would happen in the Dockerfiles right?

Reply:
Only for isos. The final system should not have that.

Reply:
on the install part, we will need to do the same for ubuntu, but in that point we know what distro is due to being able to read /etc/os-release so we can only trigger the same thing if we are on ubuntu.

Maybe we should do the same here.....